packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -595,6 +596,53 @@ describe('KeyringController', () => {
     });
   });
 
+  describe('decryptMessage', () => {
+    it('should successfully decrypt a message with valid parameters and return the raw decryption result', async () => {
+      await withController(async ({ controller }) => {
+        const { importedAccountAddress } =
+          await controller.importAccountWithStrategy(
+            AccountImportStrategy.privateKey,
+            [privateKey],
+          );
+        const message = 'Hello, encrypted world!';
+        const encryptedMessage = encrypt({
+          publicKey: await controller.getEncryptionPublicKey(
+            importedAccountAddress,
+          ),
+          data: message,
+          version: 'x25519-xsalsa20-poly1305',
+        });
+
+        const messageParams = {
+          from: importedAccountAddress,
+          data: encryptedMessage,
+        };
+
+        const result = await controller.decryptMessage(messageParams);
+
+        expect(result).toBe(message);
```

Comment:
We could also simplify this by taking inspiration from `getEncryptionPublicKey` tests, with something like this:

```suggestion
        const { importedAccountAddress } =
          await controller.importAccountWithStrategy(
            AccountImportStrategy.privateKey,
            [privateKey],
          );

        const result = await controller.decryptMessage({
          from: importedAccountAddress,
          data: {
            version: 'x25519-xsalsa20-poly1305',
            nonce: 'wLwuCz6aY+IjdF34os94LbAer8H5TPck',
            ephemPublicKey: 'B4cLIZfy4AU7xdM8DAO/0+VaIm8W2aY2xb7ciaPmpx8=',
            ciphertext:
              '4mNw0NlJfa+qWAsCza/iYk7uQtYcY/TBLLQnFsPsnS0E7ErjLU8J/ViK',
          },
        });

        expect(result).toBe('My name is Satoshi Buterin');
```

This would also remove interdependency between unit tests

Reply:
@mikesposito PR got already merged

Reply:
Ops, it got merged while I was reviewing it 😄 nevermind!